firmware/controllers/actuators/idle_thread.cpp

```diff
@@ -34,6 +34,7 @@ int IdleController::getTargetRpm(float clt) {
 
 	auto target = targetRpmByClt + targetRpmAcBump;
 	idleTarget = target;
+	target += luaAddRpm;
```

Comment:
should these two lines be swapped, so the idle logging shows the target including lua bump?

Reply:
valid point currently it wont show the lua bump anywhere